src/backend/expungeservice/endpoints/search.py

```diff
@@ -10,7 +12,7 @@
 from expungeservice.serializer import ExpungeModelEncoder
 from expungeservice.crypto import DataCipher
 from expungeservice.record_summarizer import RecordSummarizer
-from expungeservice.util import LRUCache
+from expungeservice.util import LRUCache, DateWithFuture as date_class
```

Comment:
I don't think this needs an alias.

Reply:
We're pretending like DateWithFuture is the standard `date` library, which we've called `date_class` across most of our codebase for legacy reasons. I'm not sure if I like this decision in retrospect but I'll stick with this for now until we feel like we should change it everywhere to not have an alias.